proposals/lip-0058.md

```diff
@@ -753,9 +753,9 @@ TBD
 [lip-0042]: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0042.md
 [lip-0047]: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0047.md
 [lip-0055]: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0055.md
-[lisk-bft-paper]: https://research.lisk.com/t/add-weights-to-lisk-bft-consensus-protocol/289#comparison-to-the-lisk-bft-paper-7
-[research:certificate-generation]: https://research.lisk.com/t/introduce-certificate-generation-mechanism/296
-[research:dpos-module]: https://research.lisk.com/t/define-state-and-state-transitions-of-dpos-module/320
-[research:unlock-condition]: https://research.lisk.com/t/introduce-unlocking-condition-for-incentivizing-certificate-generation/300
-[research:update-genesis-block]: https://research.lisk.com/t/update-genesis-block-schema-and-processing/325
-[research:weighted-bft]: https://research.lisk.com/t/add-weights-to-lisk-bft-consensus-protocol/289
+[lip-0056]: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0056.md
+[lip-0056-comparison-lisk-bft-paper]: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0056.md#comparison-to-the-lisk-bft-paper
```

Comment:
OCD: we use the convention # to link to a section, so I would change to:

[lip-0056#comparison-lisk-bft-paper]

Reply:
Done.